_sass/extends/_buttons.scss

```diff
@@ -0,0 +1,27 @@
+%button {
+  @include transition-property(background-color);
+  @include transition-duration(0.25s);
+  @include transition-timing-function(ease-out);
+  @include transition-delay(0s);
+  @include appearance(none);
+  display: inline-block;
```

Comment:
Properties should be ordered border, border-radius, -moz-box-sizing, -webkit-box-sizing, box-sizing, cursor, display, font-family, font-weight, line-height, line-height, margin-bottom, padding-bottom, padding-top, text-align, text-decoration, user-select, vertical-align